packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -0,0 +1,710 @@
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT",
+                    "DELETE"
+                ]
+            }
+        }
+    }
+}
+```
+
+##### Recovery Factor Profile Feature Properties
+
+| Property                  | Type                                                           | Description                                                                                               |
+| ------------------------- | -------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+| `_links`                  | [JSON HAL](https://tools.ietf.org/html/draft-kelly-json-hal-06) | Link relations for this object                                                                            |
+| `created`                 | String (ISO-8601)                                              | Timestamp when the Factor Profile feature was created.                                                    |
+| `eligiblity`              | String                                                         | Indicates if factor may be recovered. Possible values: `ALLOWED`, `NOT_ALLOWED`                           |
+| `id`                      | String                                                         | Identifier of the Factor Profile feature.                                                                 |
+| `lastUpdated`             | String (ISO-8601)                                              | Timestamp when the Factor Profile feature was last modified.                                              |
+| `type`                    | String (Enum)                                                  | Feature type.                                                                                             |
+| `verificationMethod.type` | String                                                         | Indicates which factors may be used to verify a recover operation. Possible values: `ANY_FACTOR`, `CHAIN` |
+
+###### Chain verification
+
+Chain verification may be used to specify which factor may be used to verify the recovery
+
+###### Example
+```json
+{
+    "type": "recovery",
+    "id": "fpfuidgo5kiGGK3WU0g3",
+    "created": "2019-10-08T18:19:04.000Z",
+    "lastUpdated": "2019-10-08T22:56:46.000Z",
+    "eligibility": "ALLOWED",
+    "verificationMethod": {
+        "type": "CHAIN",
+        "chains": [
+            {
+                "criteria": [
+                    {
+                        "factorProfileId": "fpruihGXyLzyGYCq30g3"
+                    }
+                ]
+            }
+        ]
+    },
+    "_links": {
+        "self": {
+            "href": "https://{yourOktaDomain}/api/v1/org/factors/{factorType}/profiles/fpruib7klOvW4pAuK0g3/features/fpfuidgo5kiGGK3WU0g3",
```

Comment:
When making examples of features not tied to specific profiles, then it would make more sense to make a generic key `{profileId}` together with `{factorType}` since the feature is agnostic to both type and profile.

Reply:
Makes sense to me.

Reply:
factorType vs `factorName`
we say `okta_password` and not password

Reply:
I'm not clear on what the action item here is.

Reply:
factorType was changed to factorName in self links everywhere in this PR (which is correct)
This spot along with 3 others where I left comments last night still shows `factorType` isntead of `factorName`

Reply:
OK, I removed all the instances of `factorType`.